docs/docsite/rst/command_guide/cheatsheet.rst

```diff
@@ -31,3 +31,26 @@ Loads ``my_playbook.yml`` from the current working directory and:
 
 See :ref:`ansible-playbook` for detailed documentation.
 
+
+ansible-galaxy
+==============
+
+Installing a collection:
+
+.. code-block:: bash
+
+   ansible-galaxy collection install mynamespace.mycollection -p /path/to/collections --force-all-deps
```

Comment:
Why not just showcase the simple `ansible-galaxy collection install` command without extra options? I don't think most users will need `-p` or `--force-all-deps`.

Reply:
Thanks! I thought maybe it was just the simple command, but I saw a few examples using the -p in the forum so wasn't sure. I can remove that stuff.